docs/source/membership/membership.md

```diff
@@ -1,341 +1,197 @@
+
+## What is an MSP?
+
+Despite its name, the Membership Service Provider does not actually provide anything. Rather, the implementation of the MSP requirement is a folder that contains a list of related identities and is used to store users's certificates or in the case of channel configuration, a list of organization. Whereas Certificate Authorities generate the certificates that represent identities, the MSP contains a list of permissioned identities.
+
+The MSP identifies which Root CAs and Intermediate CAs are accepted to define the members of a trust domain by listing the identities of their members, or by identifying which CAs are authorized to issue valid identities for their members.
+
+But the power of an MSP goes beyond simply listing who is a network participant or member of a channel. It is the MSP that turns an identity into a **role** by identifying specific privileges an actor has on a node or channel. Note that when a user is registered with a Fabric CA, a role of admin, peer, client, orderer, or member must be associated with the user.
+
+In addition, an MSP can allow for the identification of a list of identities that have been revoked --- as discussed in the [Identity](../identity/identity.html) documentation --- but we will talk about how that process also extends to an MSP.
+
+## MSP domains
+
+MSPs occur in two domains in a blockchain network:
 
-* In channel configuration (**channel MSP**)
 * Locally on an actor's node (**local MSP**)
+* In channel configuration (**channel MSP**)
+
+The key difference between local and channel MSPs is not how they function -- both turn identities into roles -- but their **scope**. Each MSP lists roles and permissions at a particular level of administration.
+
+### Local MSPs
+
+**Local MSPs are defined for clients and for nodes (peers and orderers)**.
+Local MSPs define the permissions for a node (who the peer admins are, for example). The local MSPs of clients (the account holders in the banking scenario above), allow the user to authenticate itself in its transactions as a member of a channel (e.g. in chaincode transactions), or as the owner of a specific role into the system (an org admin, for example, in configuration transactions).
+
+**Every node must have a local MSP defined**, as it defines who has administrative or participatory rights at that level (peer admins will not necessarily be channel admins, and vice versa).  This allows for authenticating member messages outside the context of a channel and to define the permissions over a particular node (who has the ability to install chaincode on a peer, for example). Note that one or more nodes can be owned by an organization. An MSP defines the organization admins. And the organization, the admin of the organization, the admin of the node, and the node itself should all have the same root of trust.
```

Comment:
I would be a bit more clear here. The node MSP not only defines the identity of that node, it also identifies have the ability to operate that node.

Reply:
`Local MSPs define the permissions for a node (who are the peer admins who can operate the node, for example).`